mangum/protocols/http.py

```diff
@@ -1,6 +1,6 @@
 import enum
 import asyncio
-from typing import Optional
+import typing
```

Comment:
Any particular motivation for this change? The import style for types should be made consistent, I think `from typing import ...` generally produces more readable code than referencing the `typing` module, but in any case it should be consistent throughout the project.

Reply:
The `lifetime.py` module had it that way and technically using `import typing` is marginally faster than `from typing import Optional` both initially and on every use of its members (idk if that's the case for all Python versions). I thought that might have been the reason for the inconsistency so decided to make it consistent in favor of plain `import`.

I'll change all of them to use `from` imports as I do agree it's more readable.